src/Sql/dbo/Dirt/Stored Procedures/OrganizationReport_Create.sql

```diff
@@ -1,10 +1,12 @@
 CREATE PROCEDURE [dbo].[OrganizationReport_Create]
     @Id UNIQUEIDENTIFIER OUTPUT,
     @OrganizationId UNIQUEIDENTIFIER,
-    @Date DATETIME2(7),
     @ReportData NVARCHAR(MAX),
     @CreationDate DATETIME2(7),
     @ContentEncryptionKey VARCHAR(MAX)
+    @SummaryData NVARCHAR(MAX),
```

Comment:
❓ There's no usage of this table yet right? If there is then you need null defaults in places like this.